### ISSUE TITLE
Keyboard inaccessible widget Lint warnings

### ISSUE THREAD

**Issue by RajaVamsi11** [openfoodfacts/openfoodfacts-androidapp#2660]
**Summary**

Android Accessibility Lint warnings contain keyboard inaccessible widgets.
We can remove Android Keyboard inaccessible widget lint warnings by adding focusable to true.
A widget that is declared to be clickable but not declared to be focusable is not accessible via the keyboard. we should add the focusable attribute as well.

There are these warnings in two XML files named `fragment_contributors.xml` and `fragment_nutrition_product.xml` which must be removed.
